### ISSUE TITLE
Input Text

### ISSUE THREAD

**Issue by x3fold** [vidkidz/vidkidz.github.io#4]
Any chance that you will be working on Input Text?

Everything else seems to work flawlessly!

**Comment by ROBERT-MCDOWELL** [comment]
Do you mean the component input text?

**Comment by x3fold** [comment]
No, just standard input text. Dynamic text is working fine but not input.

**Comment by Azshurith** [comment]
Yes I want input text to work too
